portfolio/src/main/webapp/script.js

```diff
@@ -38,16 +38,19 @@ async function checkUserAuthentication() {
 
   if (!user) {
     rightBar.innerHTML = `<a href="${loginUrl}">Login</a>`;
-    addCommentDiv.innerHTML = `<a href="${loginUrl}">Login</a> to add a comment`;
+    addCommentDiv.innerHTML = `<a href="${loginUrl}">
+        Login
+      </a> to add a comment`;
```

Comment:
Any reason why this was split into three lines?

Reply:
because if it is in one line, it would have a length of 81, which is more than 80